Modules/_sqlite/cursor.c

```diff
@@ -839,13 +888,49 @@ PyObject* pysqlite_cursor_fetchall(pysqlite_Cursor* self, PyObject* args)
     }
 }
 
-PyObject* pysqlite_noop(pysqlite_Connection* self, PyObject* args)
+/*[clinic input]
+_sqlite3.Cursor.setinputsizes as pysqlite_cursor_setinputsizes
+
+    sizes: object = None
```

Comment:
Is it optional?

Reply:
Apparently, but I find it weird. BTW, neither `setinputsizes` nor `setoutputsize` are documented.
```
$ python3.9
Python 3.9.1 (v3.9.1:1e5d33e9b9, Dec  7 2020, 12:10:52) 
[Clang 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import sqlite3
>>> con = sqlite3.connect(":memory:")
>>> cur = con.cursor()
>>> cur.setinputsizes()
>>> 
```

Reply:
They are documented in the Python DB API Spec ([PEP 249](https://www.python.org/dev/peps/pep-0249/) if there are no newer documents).

Reply:
> They are documented in the Python DB API Spec ([PEP 249](https://www.python.org/dev/peps/pep-0249/) if there are no newer documents).

That's true, but they're not documented here: https://docs.python.org/3/library/sqlite3.html

Reply:
> Apparently, but I find it weird.

On the other side, passing `None` as the `sizes` parameter is allowed according to the DB API, so defaulting to `None` is perhaps not so weird after all.

Reply:
… but the DB API actually specifies that it's not optional (no brackets around the parameters)

Reply:
According to the DB API, we should remove the default argument, but if we want to be backwards compatible, we shouldn't. But the function is a no-op, so I guess "backwards compatibility" is a no-op as well :)

Reply:
See 935140d